### PR TITLE
 Move roles from private image to base

### PR DESCRIPTION
**Pull request by MikolajK**
These were executed needlessly every time we provisioned
the private image. If they are moved to the base image
then the provision of the private image will be faster
and the public image will be better. The only problem
is that we now need to update the base image whenever
we change any of these roles.

### CODE REVIEW

File: ansible/roles/vi/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      state=directory
+      owner=vagrant
+      group=vagrant
+      mode=0775
+    with_items:
+    - { dir: '.vim' }
+
+  - name: Creates directory
+    file:
+      path=/home/vagrant/.vim/{{ item.dir }}
```
Comment (edwardmlyte) [inline ansible/roles/vi/tasks/main.yml]: can you reformat to match the `:` separated key values, like in the other roles please. just basing this on the examples given on the ansible roles docs

File: ansible/roles/bash_alias/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+  - name: Add command aliases
+    lineinfile:
+      path: /home/vagrant/.bash_aliases
+      owner: root
+      state: present
+      create: yes
+      line: "alias {{item.name}}='{{item.command}}'"
+    with_items:
+      - { name: 'pman', command: 'unset HTTPS_PROXY && unset HTTP_PROXY && /opt/Postman/Postman --disable-gpu'}
```
[the remark this answers is not in the document]
Comment (MikolajK) [reply ansible/roles/bash_alias/tasks/main.yml]: That's true, although HTTPS_PROXY is not just an env variable used by us I believe? It's very niche but I'm happy to move it somewhere else.

File: ansible/roles/bash_alias/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  - name: Add command aliases
+    lineinfile:
+      path: /home/vagrant/.bash_aliases
+      owner: root
+      state: present
+      create: yes
+      line: "alias {{item.name}}='{{item.command}}'"
+    with_items:
+      - { name: 'syncmaster', command: 'git checkout master; git fetch --all; git merge --ff-only upstream/master; git push; git checkout -'}
+      - { name: 'pr', command: 'git push origin HEAD -u && hub pull-request'}
```
Comment (edwardmlyte) [inline ansible/roles/bash_alias/tasks/main.yml]: Missed this. I reckon we should have a dependency between this role and the hub role in a future change

File: ansible/roles/vi/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      group: vagrant
+      mode: 0775
+    with_items:
+    - { src: 'vimrc', dest: '/home/vagrant/.vimrc' }
+
+  - name: run dos2unix against scripts
+    command:
+      dos2unix "{{ item }}"
+    with_items:
+    - '/home/vagrant/.vimrc'
```
Comment (edwardmlyte) [inline ansible/roles/vi/tasks/main.yml]: At some point we can get rid of these single item lists

File: ansible/roles/bash_alias/tasks/main.yml (hunk 3 above)
Comment (MikolajK) [reply ansible/roles/bash_alias/tasks/main.yml]: Yes, we need a full review of dependencies at some point.

**Pull request merged**